eng/Versions.props

```diff
@@ -25,7 +25,7 @@
   </PropertyGroup>
   <ItemGroup>
     <!-- The bands we want to produce workload manifests for -->
-    <WorkloadSdkBandVersions Include="6.0.100;6.0.200;6.0.300" />
+    <WorkloadSdkBandVersions Include="6.0.100;6.0.200;6.0.300;6.0.400" />
```

Comment:
Since the 200 band SDK is no longer being built, is this a situation where we should be removing old workloads?

Reply:
I'm not sure?  Would the customers who have a 200 SDK already installed not get the latest updates?

Reply:
It's out of support, so would they be expected to get the latest updates?

Reply:
there is some history here and I'm not sure I have it right but the way I understood things for workloads is that 200 is intended to see 100 band updates as a fallback but due to some sdk bugs we've had to keep producing multiple bands as workarounds.  Given that it seemed like we'd be matching the intended design if we if we make sure 200 still sees any changes that apply to 100.